manila/share/drivers/netapp/dataontap/client/client_cmode.py

```diff
@@ -3928,9 +3928,14 @@ def prune_deleted_volumes(self):
                 try:
                     client.send_request(
                         'volume-destroy', {'name': volume_name})
-                except netapp_api.NaApiError:
+                except netapp_api.NaApiError as e:
                     LOG.error(
                         'Pruning soft-deleted volume %s failed', volume_name)
+                    if e.code == netapp_api.EVOLDEL_NOT_ALLOW_BY_CLONE:
+                        if self.volume_clone_split_status(volume_name) != 100:
+                            self.volume_clone_split_start(volume_name)
```

Comment:
should start the split only when there is no ongoing split. this could start the split when the split is in progress, right?